src/utils/format-numbers.js

```diff
@@ -27,19 +27,10 @@ export function formatNumber(
   significantDigits: number = 2,
   maxFractionalDigits: number = 3
 ): string {
-  /*
-   * Note that numDigitsOnLeft can be negative when the first non-zero digit
-   * is on the right of the decimal point.  0.01 = -1
-   */
-  const numDigitsOnLeft = Math.floor(Math.log10(Math.abs(value))) + 1;
-  let places = significantDigits - numDigitsOnLeft;
-  if (places < 0) {
-    places = 0;
-  } else if (places > maxFractionalDigits) {
-    places = maxFractionalDigits;
-  }
-
-  return value.toFixed(places);
+  return value.toLocaleString(undefined, {
+    maximumSignificantDigits: significantDigits,
+    maximumFractionDigits: maxFractionalDigits,
+  });
```

Comment:
From [the documentation in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString): 
```
The following properties fall into two groups: minimumIntegerDigits,
minimumFractionDigits, and maximumFractionDigits in one group,
minimumSignificantDigits and maximumSignificantDigits in the other.
If at least one property from the second group is defined, then the first
group is ignored.
```

So, after all, to keep the previous behavior we'll still need to do some calculations.

This seems to work:

```js
  /*
   * Note that numDigitsOnLeft can be negative when the first non-zero digit
   * is on the right of the decimal point.  0.01 = -1
   */
  const numDigitsOnLeft = Math.floor(Math.log10(Math.abs(value))) + 1;
  let places = significantDigits - numDigitsOnLeft;
  if (places < 0) {
    places = 0;
  } else if (places > maxFractionalDigits) {
    places = maxFractionalDigits;
  }

  return value.toLocaleString(undefined, {
    minimumFractionDigits: places,
    maximumFractionDigits: places,
  });
```